### PR TITLE
feat(docs): FRMW-200 - Migrate rxToggle docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-200

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxToggle/README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
-This component provides and attribute to handle toggling a boolean scope property for hide/show purposes. Normally used in conjunction with ng-show to toggle hidden content. See the collapse functionality in 'rxApp' for a real-world use.
-
-## Future plans
-
-In conjunction with rxToggle, it would be helpful to have an attribute that binds the visibility state of an element with an event, so that when an event is fired from a component of the same type, it hides all other components of that time. For example, if a pop-up menu appears on click of a 'gear' component, it should hide any other existing pop-up menus currently showing.
\ No newline at end of file
+This component provides and attribute to handle toggling a boolean scope
+property for hide/show purposes. Normally used in conjunction with ng-show to
+toggle hidden content. See the collapse functionality in 'rxApp' for a
+real-world use.
```
Comment (CITguy) [inline src/rxToggle/README.md]: * Grammar is strange
* Content out of date
* Some misspellings

I'd go with:

```
This component provides an attribute directive to handle 
toggling a boolean scope property for show/hide purposes 
(normally used in conjunction with ng-show to toggle hidden content).
```

File: src/rxToggle/rxToggle.js [hunk 2]
```diff
@@ -4,7 +4,19 @@
  * @description
  * # rxToggle Component
  *
- * [TBD]
+ * This component provides and attribute to handle toggling a boolean scope
+ * property for hide/show purposes. Normally used in conjunction with ng-show
+ * to toggle hidden content. See the collapse functionality in 'rxApp' for a
+ * real-world use.
```
Comment (CITguy) [inline src/rxToggle/rxToggle.js]: Same as README

File: src/rxToggle/README.md (hunk 1 above)
Comment (bry4n) [inline src/rxToggle/README.md]: updated.

File: src/rxToggle/rxToggle.js [hunk 3]
```diff
@@ -4,7 +4,19 @@
...
+ *
+ * ## Future plans
+ *
+ * In conjunction with rxToggle, it would be helpful to have an attribute that
+ * binds the visibility state of an element with an event, so that when an
+ * event is fired from a component of the same type, it hides all other
+ * components of that time. For example, if a pop-up menu appears on click of
+ * a 'gear' component, it should hide any other existing pop-up menus currently
+ * showing.
```
Comment (CITguy) [inline src/rxToggle/rxToggle.js]: Remove "Future plans" section. This has been captured as a feature request in [FRMW-257](https://jira.rax.io/browse/FRMW-257).

Comment (bry4n) [inline src/rxToggle/rxToggle.js]: updated

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**